### PR TITLE
Types for leaflet-side-by-side

### PR DESCRIPTION
**Pull request by OliverBiel**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`

### CODE REVIEW

File: types/leaflet-side-by-side/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// Type definitions for react-side-effect 2.2
+// Project: https://github.com/digidem/leaflet-side-by-side
+// Definitions by: Remo H. Jansen <https://github.com/OliverBiel/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
```
Comment (OliverBiel) [inline types/leaflet-side-by-side/index.d.ts:4]: Forgot to change the name of the package (Leaflet Side by Side) and my name (Gabriel O. Martins)

Comment (peterblazejewicz) [reply types/leaflet-side-by-side/index.d.ts:4]: so can you make this change? Thanks!

File: types/leaflet-side-by-side/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
+// Type definitions for leaflet-side-by-side 2.2
+// Project: https://github.com/digidem/leaflet-side-by-side
+// Definitions by: Gabriel O. Martins <https://github.com/OliverBiel/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// Minimum TypeScript Version: 5.0
```
Comment (peterblazejewicz) [inline types/leaflet-side-by-side/index.d.ts:6]:
```suggestion
```
IMO this is not needed (there is no language features from TS5.0 that would require this constraint)

**Comment by peterblazejewicz** [comment]
👍🏻

**Pull request merged**